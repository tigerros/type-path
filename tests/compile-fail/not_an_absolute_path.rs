use type_path::type_path;

fn main() {
    type_path!(std::io::BufWriter);
}
