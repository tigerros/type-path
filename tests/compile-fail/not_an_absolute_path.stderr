error: expected `::` or `crate`
 --> tests/compile-fail/not_an_absolute_path.rs:4:16
  |
4 |     type_path!(std::io::BufWriter);
  |                ^^^
